plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -125,6 +125,44 @@
               version="1.22.0" />
       </libraryFile>
     </library>
+    
+    <library
+          id="googlecloudcore"
+          group="clientapis"
+          name="Google Cloud Core Client Library for Java"
+          tooltip="%googlecloudcore.tooltip"
+          siteUri="https://googlecloudplatform.github.io/google-cloud-java/" >
+      <libraryDependency id="googleapiclient" />
+      <libraryFile
+            javadocUri="https://googlecloudplatform.github.io/google-cloud-java/0.10.0/apidocs/">
+        <mavenCoordinates
+              artifactId="google-cloud-core"
+              groupId="com.google.cloud"
+              version="0.10.0-alpha" />
+      </libraryFile>
+      <libraryFile javadocUri="http://www.joda.org/joda-time/apidocs/">
+        <mavenCoordinates
+              artifactId="joda-time"
+              groupId="joda-time"
+              version="2.9.2" />
+      </libraryFile>
+    </library>
+    
+    <library
+          id="googlecloudstorage"
+          group="clientapis"
+          name="Google Cloud Storage Client Library for Java"
```

Comment:
I think it's a proper name. Do we localize "Google Cloud Platform" for example? Let me check with the relevant team.